### ISSUE TITLE
ERROR: Cannot install tt-smi and tt-smi==2.1.0 because these package versions have conflicting dependencies.

### ISSUE THREAD

**Issue by milank94** [tenstorrent/tt-smi#13]
Hitting this error when trying to install `tt-smi` from main ([c943e04](https://github.com/tenstorrent/tt-smi/commit/c943e047f6237e3aff1c86c6a0b42014b1930a4d))

Followed these steps:
```
python3 -m venv .venv
source .venv/bin/activate
pip3 install --upgrade pip
pip3 install .
```

![image](https://github.com/tenstorrent/tt-smi/assets/78221808/4a71cce7-7b2e-4aba-b0d7-0810d2f87dbb)

**Comment by sbansalTT** [comment]
can you try again? this should be fixed as of v2.2.0

**Comment by milank94** [comment]
@sbansalTT Thank you, that worked!
